scripts/run-testchain.sh

```diff
@@ -3,5 +3,5 @@ set -e
 
 CWD="${0%/*}"
 
-$CWD/set-polling-interval.sh
```

Comment:
Not sure if this should be left commented out either. Flagging to review as well.

Reply:
it was making a change to a file in ethers that no longer exists, but I'll double check if I can change the path to the file it was looking for.